plugins/ruby-foreman-tasks/debian/control

```diff
@@ -2,7 +2,7 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.7)
+Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.9)
```

Comment:
If that package should really forcefully depend on dynflow 0.7.9 or later, the ruby-foreman-tasks version must indeed be bumped to -4, but as foreman-tasks 0.6.13 works with dynflow 0.7.7 also, I don't really a point in doing that.